src/main/kotlin/no/nav/familie/ks/sak/kjerne/regelverk/RegelverkUtleder.kt

```diff
@@ -0,0 +1,13 @@
+package no.nav.familie.ks.sak.kjerne.regelverk
+
+import java.time.LocalDate
+
+object RegelverkUtleder {
+    val FØDSELSDATO_GRENSE_LOVENDRING_FEBRUAR_2025 = LocalDate.of(2024, 1, 1)
```

Comment:
Tenker at denne datoen kan være forvirrende for en som ikke jobber med dette. En doc om hva denne datoen er med eksempler hadde hjulpet f.eks meg.

Reply:
Den nye lovendringen blir mest sannsynlig gjeldene for alle barn født etter en viss dato. På praten forrige uke ble det slengt ut at 1.1.24 kan være en potensiell kandidat hvis lovendringen trer i kraft i jan/feb 25. Har lagt inn en todo i koden nå på at denne datoen må oppdateres når endelig lov faktisk blir vedtatt, for enn så lenge er det synsing fra vår side 😅

Reply:
Var egentlig ikke datoen jeg kommenterte på, men lesbarhet om 1 til 2 år når vi har glemt alt. En enum-verdi er ikke alltid lesbart.

Reply:
Skjønner! Tenker vi kan legge på bedre dokumentasjon når endelig regelverk er vedtatt 😊